.github/workflows/build.yaml

```
@@ -71,34 +71,30 @@ jobs:
         uses: azure/setup-kubectl@v3
         with:
           version: v1.21.2
-      - 
-        name: Create KUBECONFIG
-        env:
-          KUBE_CONFIG: '${{ secrets.KUBECONFIG_TEST }}'
-        run: |
-          mkdir -p $HOME/.kube
-          echo "$KUBE_CONFIG"
-          echo "$KUBE_CONFIG" > $HOME/.kube/config
       - 
         name: Deploy Helm Release
         env:
           TRAINING_HELM_RELEASE: 'pr-${{ github.event.pull_request.number }}'
-          TRAINING_NAMESPACE: 'pitc-ansible-training-test'
+          TRAINING_NAMESPACE: 'pitc-ansible-training-stg'
           TRAINING_VERSION: '${{ github.sha }}'
+          KUBE_TOKEN: '${{ secrets.OPENSHIFT_SA_TOKEN_STG }}'
+          KUBE_URL: '${{ secrets.OPENSHIFT_API_URL }}'
         run: |
-          helm upgrade $TRAINING_HELM_RELEASE acend-training-chart --install --wait --kubeconfig $HOME/.kube/config  --namespace=$TRAINING_NAMESPACE --set=app.name=$HELM_RELEASE --set=app.version=$TRAINING_VERSION --repo=https://acend.github.io/helm-charts/ --values=helm-chart/values.yaml --atomic
+          helm upgrade $TRAINING_HELM_RELEASE acend-training-chart --debug --install --wait --kube-apiserver=$KUBE_URL --kube-token=$KUBE_TOKEN --namespace=$TRAINING_NAMESPACE --set=app.name=$HELM_RELEASE --set=app.version=$TRAINING_VERSION --repo=https://acend.github.io/helm-charts/ --values=helm-chart/values_openshift.yaml --atomic
```

I'd recommend not renaming `values.yaml`, either leave it as is or at least copy it so the old values could be easily looked up.

```suggestion
          helm upgrade $TRAINING_HELM_RELEASE acend-training-chart --debug --install --wait --kube-apiserver=$KUBE_URL --kube-token=$KUBE_TOKEN --namespace=$TRAINING_NAMESPACE --set=app.name=$HELM_RELEASE --set=app.version=$TRAINING_VERSION --repo=https://acend.github.io/helm-charts/ --values=helm-chart/values.yaml --atomic
```

---

applied